CMakeLists.txt

```diff
@@ -248,6 +248,19 @@ set_target_properties(rdf4cpp PROPERTIES
         CXX_STANDARD_REQUIRED ON
 )
 
+option(DBPEDIA_MODE "relaxed parsing for dbpedia" OFF)
```

Comment:
We probably won't need a compiletime switch. It will most probably always be set to runtime. Given how people usually handle data they probably don't want a seperate executable/compiled library for dbpedia.